src/__tests__/__mocks__/mockCoins.js

```diff
@@ -0,0 +1,157 @@
+// Mock Data
+export const mockCoins = [
+  {
+    id: '90',
+    symbol: 'BTC',
+    name: 'Bitcoin',
+    nameid: 'bitcoin',
+    rank: 1,
+    price_usd: '25767.96',
+    percent_change_24h: '0.25',
+    percent_change_1h: '0.06',
+    percent_change_7d: '-7.84',
+    price_btc: '1.00',
+    market_cap_usd: '501726497779.56',
+    volume24: 7226105206.242357,
+    volume24a: 7356567443.02128,
+    csupply: '19470946.00',
+    tsupply: '19470946',
+    msupply: '21000000',
+  },
+  {
```

Comment:
- [ ] nice work. Please kindly note that this was a mobile version project, and we were supposed to follow the design to the fullest while personalizing the content. Kindly make the grid color apply to the smaller screen. You have just one plain color instead of thick and light as in your desktop and wider screens. Please implement this, so it can match with the original design. 
- [ ] please kindly remove the white lines separating the cards like in the original design.
- [ ] kindly add the mobile size as it switches to the desktop version too early ie 550px .. Kindly work on that, having in mind that it was a mobile app. nice work.

yours
![Screenshot_16](https://github.com/hamidazim321/coin-watch/assets/65298048/2c8f6a43-762d-4350-b471-6807f0956e55)
original
![Screenshot_17](https://github.com/hamidazim321/coin-watch/assets/65298048/ffd64fb6-a6bf-45f4-ac7c-26c7c6a5d277)